### ISSUE TITLE
BG2: Using the scroll wheel in the levelup screen is unintuitive

### ISSUE THREAD

**Issue by Laulajatar** [gemrb/gemrb#1456]
#### Description
Using the scroll wheel in the levelup screen always scrolls the proficiency slots, except when exactly on the scrollbar of the skills. 

It would make more sense for the skills to move when scrolling when the cursor is over their side of the window. 

#### Additional context
In the gif I am using my scroll wheel. 
![Peek 2021-08-23 22-14](https://user-images.githubusercontent.com/68446748/130521537-174c46af-2339-43f6-ab6f-35ef77fa58a4.gif)

**Comment by lynxlynxlynx** [comment]
We have the capabilities to fix this, iirc using a hacked proxy control (GUISTORE?).

**Comment by bradallred** [comment]
yeah, its not perfect tho. it only works well in GUISTORE because labels ignore events. This window has buttons so they still eat the scroll wheel. Its easy to have the buttons that are uses as labels do the same, but the +/- buttons cannot.

**Comment by lynxlynxlynx** [comment]
Yeah, there are both buttons and labels in the skill name part. I think it's easiest if we deviate from the original here, disable the buttons, have labels instead display the text on mouse over and be done with it. We already add a scrollbar proxy as well.

**Comment by bradallred** [comment]
I've already generalized this capability in #1481

**Comment by lynxlynxlynx** [comment]
Ok, let's wait for that work to be finished then.
